oaknational/Oak-Web-Application
# LESQ-398 fix(download-existence): transform legacy download response to match …

…new dta schema

transform legacy dowload existence res to match new res schema

- List of changes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2027--oak-web-application.netlify.thenational.academy/teachers/programmes/design-technology-primary-ks2-l/units/cooking-and-nutrition-healthy-and-varied-diets-ec15/lessons/designing-for-a-target-market-64tkgt/downloads?preselected=worksheet
2. Click on dev tools
3. You should see no getResourceExistence Error anymore

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
